packages/browser/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@segment/analytics-next",
-  "version": "1.38.0",
+  "version": "1.39.0",
```

Comment:
Per my understanding of the release process, we don't want to bump the version here.

Reply:
This is temporary. Will be reverted right before merge.